WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3475,10 +3477,10 @@ e_byzantium = {
 			independent = yes
 			OR = {
 				culture = helleno_aramean
-				culture = greek
+				culture_group = byzantine
```

Comment:
The AI can only form the de jure kingdom under retarded and bizzare restrictions, but somehow, players are example in some cases, which is dumb.

In addition, there was, at one point, a retarded Prestige requirement that needed something like twenty thousand Prestige for the AI to create a Custom Kingdom, and even more for an Empire, instead of using the fair, balanced, Vanilla rendition's values.